docs/src/userguide/subsetting_a_cube.rst

```diff
@@ -10,6 +10,13 @@ However it is often necessary to reduce the dimensionality of a cube down to som
 Iris provides several ways of reducing both the amount of data and/or the number of dimensions in your cube depending on the circumstance.
 In all cases **the subset of a valid cube is itself a valid cube**.
 
+
+.. seealso::
+
+    Relevant gallery examples:
+    :ref:`sphx_glr_generated_gallery_general_plot_polynomial_fit.py` (Slices)
+    :ref:`sphx_glr_generated_gallery_general_plot_anomaly_log_colouring.py` (Extraction)
```

Comment:
```suggestion
    Relevant gallery examples:
      - :ref:`sphx_glr_generated_gallery_general_plot_polynomial_fit.py` (Slices)
      - :ref:`sphx_glr_generated_gallery_general_plot_anomaly_log_colouring.py` (Extraction)
```

Reply:
@ESadek-MO Minor tweak to render these two links as:
![image](https://user-images.githubusercontent.com/2051656/201073581-da521064-b373-4917-bb19-cf2517ff236b.png)
Instead of:
![image](https://user-images.githubusercontent.com/2051656/201073969-baf8b5cb-e094-4ead-b99c-2dbdc9426d0b.png)